src/data/Thread.js

```diff
@@ -97,8 +97,8 @@ class Thread {
         for (const [i, chunk] of chunks.entries()) {
           // Only send embeds, files, etc. with the last message
           const msg = (i === chunks.length - 1)
-            ? await bot.createMessage(this.channel_id, chunk, file)
-            : await bot.createMessage(this.channel_id, chunk);
+            ? await bot.createMessage(this.channel_id, { ...content, content: chunk }, file)
+            : await bot.createMessage(this.channel_id, { ...content, content: chunk });
```

Comment:
`content` can also be a string here. Could turn it into an object above first using something like:
```js
const contentObj = (typeof content === "string" ? {} : content);
```

Reply:
Good catch